objects_generated.go

```diff
@@ -21846,7 +21846,7 @@ type ZoneDelegated struct {
 	// This provides information for the remote name server that maintains data for
 	// the delegated zone. The Infoblox appliance redirects queries for data for
 	// the delegated zone to this remote name server.
-	DelegateTo []NameServer `json:"delegate_to,omitempty"`
+	DelegateTo NullForwardTo `json:"delegate_to,omitempty"`
```

Comment:
Update this for NullNameserver as ForwardTo is not a appropriate keyword for DelegateTo field